controller/cache/admission.go

```diff
@@ -773,22 +773,28 @@ func doesMapContain(key, value string, propMap map[string]string) bool {
 	return false
 }
 
-// does propKey=propValue matches kvMap(criteria)
-func doesPropMatchCrt(propKey, propValue string, kvMap map[string]string) bool {
-	if value, exist := kvMap[propKey]; exist {
-		// kvMap contains propKey
-		if value != "" {
-			// does kvMap contain propKey=propValue?
-			return share.EqualMatch(value, propValue)
+
+func doesCrtContainProp(propKey, propValue string, kvMap map[string][]string) bool {
```

Comment:
Is this line correct?
Let's say:
propKey="DESCRIPTION"
propValue=""
kvMap={"DESCRIPTION": {"", "abc", "iperfserverb"}}
This function will return false.
But is it expected to return false or true in this case?

Reply:
Will the `propValue` argument ever be an empty value? It is sourced from the `nvsysadmission.AdmResObject.Labels` and `nvsysadmission.ScannedImageSummary.Labels` fields.

Reply:
After discussion, added behavior to return true for empty propValues when propKey is matched.